common/src/watchMain/kotlin/com/surrus/common/Dispatcher.kt

```diff
@@ -0,0 +1,19 @@
+package com.surrus.common
+
+import kotlin.coroutines.*
+import kotlinx.coroutines.*
+import platform.darwin.*
+
+
+internal actual val ApplicationDispatcher: CoroutineDispatcher =
```

Comment:
Added this actual which is the same as the iOS actual.

Reply:
With addition of the native coroutines dependency these (`Dispatcher.kt`) were no longer needed.  I guess fork was probably created before that change was made?